curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/accessing-nested-objects.md

```diff
@@ -48,13 +48,15 @@ assert(gloveBoxContents === 'maps');
 Your code should use dot and bracket notation to access `myStorage`.
 
 ```js
-assert(/=\s*myStorage\.car\.inside\[\s*("|')glove box\1\s*\]/g.test(code));
+assert(/=\s*myStorage\.car\.inside\[\s*("|')glove box\1\s*\]|const\s*{\s*'glove box':\s*gloveBoxContents\s*}\s*=\s*myStorage\.car\.inside;/g.test(code));
```

Comment:
```suggestion
assert.match(code, /myStorage\.car\.inside);
```

- Use the proper assert API
- Since we don't really care if they do `["glove box"]` or use destructuring, we might as well only test the property chain access.

Reply:
done